amclin/aem-packager
# fix: configurations not loading from package.json

- Fixes : Package gets created in target folder instead of custom path provided

Fixes #486

## Proposed Changes

- Call the method `getConfigsFromPackage` instead of passing the identifier

## Screenshots and Logs 

### Before

Last line from log : AEM package has been created and can be found in the current user's Maven package repository or in ./target

### After

Last line from log : AEM package has been created and can be found in the current user's Maven package repository or in ./build-aem/package

## Background context

...fill in details explaining why this is important or this approach should be used to solve the issue...

### Where should the reviewer start / What requires special attention?

...specific files or actions...

### How should this be manually tested?

1. Create a new AEM project using react as frontend module type
2. Include `aem-packager` in package.json dev dependencies and then use entries below for `aem-packager` section for ui.frontend module
3. Update build script as `react-scripts build && clientlib && aem-packager`
4. Execute the command `npm i` followed by `npm build`
5. You shall see that package is created in target folder before this PR and will be the custom path mentioned in sample config section share in step2

### Questions:
- Does this add new dependencies (node, maven, ant, etc) that must be installed?
  -No
- Does this change the commands required to run this project?
  -No
- Are any environmental dependencies or operational support steps (sql commands, config changes, etc) required?
  -No
- Are there any relevant issues besides this one?
  - No
